src/decode.rs

```diff
@@ -3174,10 +3192,11 @@ fn decode_b(
         let b4_stride = usize::try_from(f.b4_stride).unwrap();
         let cur_segmap = &f.cur_segmap.as_ref().unwrap().inner;
         let offset = by * b4_stride + bx;
-        CaseSet::<32, false>::one((), bw4, 0, |case, ()| {
+        case_set!(32, len = bw4, case_offset = 0, {
```

Comment:
Why `case_offset` instead of `offset` here?

Reply:
`case` would shadow a variable here, which we don't want.

Reply:
What `case` variable?

Reply:
Oh, you mean `offset` variable.

Reply:
Sorry, yep